### ISSUE TITLE
issue about voltage in

### ISSUE THREAD

**Issue by HongweiQin** [pololu/l3g-arduino#11]
Hi,

First of all, thank you for your library.

I tested it with my arduino nano, it works well.

However, there might be some mistakes on your README.

I followed your connection advice to connect VCC to +5V, it all goes well until I accidentally touched my arduino nano board.

It was really hot! Thanks god I wasn't scalded. Moreover, the light on my arduino nano turns from green to orange.

After reading the L3G4200D datasheet, I noticed that its recommended voltage is 2.4V-3.6V. I used 3.3V to retest it. It works fine.

As a result, I strongly recommend you to modify your README and to tell users to refer to their own sensor board's datasheet to determine the correct input voltage.

Thank you

**Comment by DavidEGrayson** [comment]
This library's README does not have "VCC" in it.  Why did you connect VCC to +5V?

**Comment by HongweiQin** [comment]
@DavidEGrayson 
Thanks for reply. My board is GY50. I googled "arduino GY50" to find a library for my board. I found a blog telling me to use your library. So I came here.
My board doesn't contain a VIN pin, and it use VCC pin to supply power.
I checked GY50's schematic diagram just now. It does contain a voltage converter from +5V to 3.3V. So I can't find what the problem is.
Not until just now did I realize that your library is built aiming at pololu products. Pololu products may not have such kind of issue. My apologize.
